base/gcutils.jl

```diff
@@ -109,6 +109,8 @@ Module with garbage collection utilities.
 """
 module GC
 
+public gc, enable, @preserve, safepoint, enable_logging, logging_enabled
```

Comment:
I think `enable_logging` and `logging_enabled` should maybe not have been made public?

I understand public to be part of the API guarantee?